README.md

```diff
@@ -142,6 +142,8 @@ Create an NFS server with storage:
 
     KETREW_CONFIG=./ketrewdocker/configuration.ml ocaml nfs_server.ml up submit
 
+If you'd like this NFS pool mounted on the cluster you're about to create, you should edit your configuration.env to add it to the CLUSTER_NFS_MOUNT list; stratotemplate cannot do this automatically for you.
```

Comment:
It "could" but we don't want that.

It could be a good occasion to mention that the storage in on `/nfs-pool` and the witness is `.stratowitness` on the newly created servers.

Also please wrap at 80 chars :)